client/src/components/FormAction/FormAction.scss

```diff
@@ -80,7 +80,6 @@
 }
 
 // Used buttons with text and icons, but you wan to hide the text only. eg. gridfield pagination
-// Could change to BS .text-hide { @include text-hide(); }, although slightly different
```

Comment:
`.text-hide` changed and this comment seems kinda useless anyway ("although slightly different" - so we can't change it to that then?) so removed instead of updated